06-blockchain/03-subnet.md

```diff
@@ -1,161 +0,0 @@
-### subnet contract
```

Comment:
hm, it is really a problem, dont we want to have some CI step for such a check?

Reply:
Too rare to care for now.